tina/schema.tsx

```diff
@@ -481,5 +482,38 @@ export const schema = defineSchema({
         },
       ],
     },
+    {
+      name: 'meetingLinks',
+      label: 'Meeting Links',
+      path: 'content/meeting-links',
+      format: 'json',
+      fields: [
+        {
+          type: 'object',
+          label: 'Meeting List',
+          name: 'meetingList',
+          list: true,
+          fields: [
+            {
+              name: 'bookingCard',
+              label: 'Booking Card',
+              type: 'object',
+              list: true,
+              ui: {
+                itemProps: (item) => ({
+                  label: item.name,
+                }),
+              },
+              fields: [
+                { name: 'name', label: 'Name', type: 'string' },
+                { name: 'description', label: 'Description', type: 'string' },
+                { name: 'image', label: 'Image', type: 'image', description: 'Image headshot for a meeting card' },
+                { name: 'url', label: 'URL', type: 'string' },
```

Comment:
Add some info about what kind of URL (hubspot meeting booking link

(Tip: Always assume your editors are silly geese, give them lots of tips).

Reply:
🪿